### ISSUE TITLE
"Quality" data sources

### ISSUE THREAD

**Issue by pl31** [jgehrcke/covid-19-germany-gae#31]
Best I could find (thx Berliner Morgenpost):

Thüringen:
- https://www.landesregierung-thueringen.de/corona-bulletin

Schleswig-Holstein:
- https://www.schleswig-holstein.de/DE/Landesregierung/I/Presse/_documents/Corona-Liste_Kreise.html

Sachsen-Anhalt:
- https://ms.sachsen-anhalt.de/themen/gesundheit/aktuell/coronavirus/

**Comment by jgehrcke** [comment]
Yes yes yes! I also like these. Please have a look at the README in my repository here, section https://github.com/jgehrcke/covid-19-germany-gae#quality-data-sources-published-by-bundesl%C3%A4nder
